velox/row/UnsafeRowDeserializers.h

```diff
@@ -552,6 +552,40 @@ struct UnsafeRowPrimitiveBatchDeserializer {
  * a Vector.
  */
 struct UnsafeRowDeserializer {
+ public:
+  /// Deserialize the rows which is serialized in contiguous memory.
+  /// Deserialize fast when all the column types are primitive type, otherwise,
+  /// deserialize rows one by one, as previously. Fast deserialization
+  /// computes the start memoryAddress of one data by the column index and row
+  /// number, and set all the data of one column at once.
```

Comment:
Comment suggestion:
> /// Deserializes rows that are stored in contiguous memory.
/// If all column types are primitive, fast deserialization is used.
/// Otherwise, rows are deserialized one by one as before.
/// Fast deserialization calculates the starting memory address of data
/// based on the column index and row number, setting all data for one column at once.